docs/modules/ROOT/pages/governance.adoc

```diff
@@ -319,3 +320,105 @@ await governor.execute(
 ```
 
 Executing the proposal will transfer the ERC20 tokens to the chosen recipient. To wrap up: we set up a system where a treasury is controlled by the collective decision of the token holders of a project, and all actions are executed via proposals enforced by on-chain votes.
+
+== Timestamp based governance
+
+=== Motivation
+
+It is sometimes difficult for humans to deal with durations expressed in number of blocks because of inconstistent or unpredictable time between blocks. That is particularly true of some L2 networks where blocks are being produced based on blockchain usage. Using number of blocks can also lead to the governance rules being affected by network upgrades eventually modifying the mean time between blocks.
```

Comment:
```suggestion
It is sometimes difficult to deal with durations expressed in number of blocks because of inconsistent or unpredictable time between blocks. This is particularly true of some L2 networks where blocks are produced based on blockchain usage. Using number of blocks can also lead to the governance rules being affected by network upgrades that modify the expected time between blocks.
```